pkg/operator/ceph/cluster/watcher.go

```diff
@@ -262,11 +262,20 @@ func listRBDPV(listPVs *corev1.PersistentVolumeList, cluster *cephv1.CephCluster
 	var listRbdPV []corev1.PersistentVolume
 
 	for _, pv := range listPVs.Items {
-		// Ignore PVs that support multinode access (RWX, ROX), since they can be mounted on multiple nodes.
-		if pvSupportsMultiNodeAccess(pv.Spec.AccessModes) {
+		// Skip if pv is not provisioned by CSI
+		if pv.Spec.CSI == nil {
+			logger.Warningf("pv %q is not provisioned by CSI", pv.Name)
 			continue
 		}
+
 		if pv.Spec.CSI.Driver == fmt.Sprintf("%s.rbd.csi.ceph.com", cluster.Namespace) {
+			if pvSupportsMultiNodeAccess(pv.Spec.AccessModes) {
+				continue
+			}
+			if pv.Spec.CSI.VolumeAttributes["staticVolume"] == "true" || pv.Spec.CSI.VolumeAttributes["pool"] == "" || pv.Spec.CSI.VolumeAttributes["imageName"] == "" {
+				logger.Warningf("skipping, static pv %q", pv.Name)
```

Comment:
I think info is most appropriate here. I think warnings are generally best avoided. Usually, something should be an error if the user to fix something, otherwise an info message. Warnings can cause concern or annoyance for users who watch the logs. More likely, most users simply won't see the log message, and a true warning should be raised via a status item or an emitted event. I don't see a need to set a status or emit an event for this; thus, I think info is best.

```suggestion
				logger.Infof("skipping, static pv %q", pv.Name)
```

Reply:
make sense. Thanks for comment